gnovm/pkg/gnolang/nodes.go

```diff
@@ -1717,6 +1839,32 @@ func (sb *StaticBlock) getLocalIsConst(n Name) bool {
 	return false
 }
 
+func (sb *StaticBlock) IsAssignable(store Store, n Name) bool {
```

Comment:
In this particular scenario（prohibit assign to func itself), the name `IsAssignable` is overly generalized, there are already `checkAssignableTo` in the codebase. So I guess some other name would be better.

Aside, the func should not be addressable either, but the below code works:
```go
package main

func foo() {
	p := &foo
	println(p)
}

func main() {
	foo()
}
```